### DISCUSSION TITLE
Android version

### DISCUSSION THREAD

**Discussion by trymeouteh** [jely2002/youtube-dl-gui#343] [Ideas]
An Android app to download audio and videos from many different websites. This way someone on the go can download a audio/video file and transfer it later to their phone.

This could be done in the future once Tauri supports mobile devices.

https://tauri.studio/

**Comment by jely2002** [comment]
This is out of scope for this app. I don't plan on adding a feature like this as there's quite some challenges, its not just a simple port.
